bigquery/README.md

```diff
@@ -16,7 +7,12 @@ source .bashrc
 ./google-cloud-sdk/bin/gcloud init
 ```
 
-Load the data:
+Create the dataset and table in BigQuery:
+```
+./create.sh
```

Comment:
I followed the steps on l. 3-7 (all successful), then `./create.sh` returns this:

```
/data/ClickBench/bigquery (bigquery_main %=) $ ./create.sh
BigQuery error in mk operation: Cannot determine dataset described by test
BigQuery error in query operation: Cannot start a job without a project id.
```

What do I need to fix this?

Reply:
Did `gcloud init` complete successfully? Looks like the default project id not set for this using `gcloud init`.